editor/assets/chunks/legacy/shading-standard-base.chunk

```diff
@@ -150,20 +150,22 @@ vec4 CCStandardShadingBase (StandardSurface s, vec4 shadowPos) {
   #endif
   
   #if CC_USE_LIGHTMAP != LIGHT_MAP_TYPE_ALL_IN_ONE
```

Comment:
replace CC_USE_LIGHTMAP judgement with CC_RECEIVE_DIRECTIONAL_LIGHT
surface shader needs to be modified as well